Vagrantfile

```diff
@@ -1,8 +1,50 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+require 'yaml'
+AWS = YAML.load_file 'aws.yml'
+
+puts ""
+
+if Vagrant::Util::Platform.windows?
+    puts "Vagrant was launched from Windows."
+elsif Vagrant::Util::Platform.mac?
```

Comment:
doesn't work on my mac. maybe switch to .cygwin?

Reply:
you probably want to switch the 'mac' with 'darwin'

Reply:
can you try:
puts Vagrant::Util::Platform

Reply:
I have used 'puts Vagrant::Util::Platform.platform'. The output is: darwin13.4.0

Reply:
removed the "Vagrant was launched from..."